helpers/templates/time_test.go

```diff
@@ -139,3 +140,103 @@ func TestHumanizeDurationSecondsInt(t *testing.T) {
 		})
 	}
 }
+
+func TestHumanizeTimestampInt(t *testing.T) {
+	tc := []struct {
+		name     string
+		input    int
```

Comment:
I've noticed that this part here is equal on all new tests:
```go
			result, err := HumanizeTimestamp(tt.input)
			require.NoError(t, err)
			require.Equal(t, tt.expected, result)
```

If we change the input to `interface`
```suggestion
		input    interface{}
```

Could we merge all those tests into a single one?

Reply:
Makes sense, will try doing it a bit later

Reply:
@ArthurSens done